docs/user-guide/tiramisu/installing-payloadloader.md

```diff
@@ -14,11 +14,12 @@ Installing the PayloadLoader will let you access Tiramisu just by running the He
 
 1. Take the SD Card out of your PC and plug it into your Wii U.
 1. Execute the web browser exploit as [previously explained](browser-exploit) but this time, you will need to hold the X button to open the Environment Loader menu.
-1. Navigate the list using the D-Pad and navigate to the `installer` environment, press A to launch it.
-1. Press the A button to check if you can install the PayloadLoader.
+    - As explained before, if it freezes or nothing happens, reboot the console, [reset the browser's save data](https://en-americas-support.nintendo.com/app/answers/detail/a_id/1507/~/how-to-delete-the-internet-browser-history) and try again.
+3. Navigate the list using the D-Pad and navigate to the `installer` environment, press A to launch it.
```

Comment:
Keep all of the steps as `1.`. Docsify will order them automatically.

Reply:
Not sure what happened here (I didnt even change the number orders). I'll fix it as soon as possible!